### PR TITLE
feat(slack): Add icon / team name

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This Implementation doesn't do anything to refresh the icon, so if there's no icon (slack sends back a default iocn) initially then there will never be an icon unless they re-add the integration.

### CODE REVIEW

**Review by tkaemming — DISMISSED** [review]
Travis doesn't like it.

**Comment by EvanPurkhiser** [comment]
Yeah fixing now, travis wants a mock response

File: src/sentry/integrations/slack/integration.py [hunk 1]
```diff
@@ -53,11 +54,24 @@ def get_pipeline_views(self):
 
         return [identity_pipeline_view]
 
+    def get_team_info(self, access_token):
+        payload = {
+            'token': access_token,
+        }
+
+        session = http.build_session()
+        resp = session.post('https://slack.com/api/team.info', data=payload)
```
Comment (mattrobenolt) [inline src/sentry/integrations/slack/integration.py]: I like that this is a POST and not a GET. Going to blindly trust that this is correct.

**Review by mattrobenolt — APPROVED** [review]
> This Implementation doesn't do anything to refresh the icon

Is it worth following up on a periodic job to refresh integration metadata? I assume icon isn't the only thing that can change.

File: src/sentry/integrations/slack/integration.py (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/integrations/slack/integration.py]: I do enjoy that `POST` works.. but looking at the docs it does say "Preferred method `GET`". 2480fb0

**Pull request merged**